packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -223,7 +223,7 @@ export interface TableOptions extends SchemaOptions {
    * This property can only be set if `encryption` is set to `TableEncryption.CUSTOMER_MANAGED`.
    *
    * @default - If `encryption` is set to `TableEncryption.CUSTOMER_MANAGED` and this
-   * property is undefined, a new KMS key will be created and associated with this table.
+   * property is undefined, the table will not be encrypted.
```

Comment:
One of these things is not like the other.

- In the other properties, the `@default` describes what DynamoDB does by default.
- In this property, the `@default` describes what *our code* does by default.

It could be that we're wrong about the first, but I'm pretty sure we're not wrong about the second. Or if we are, then that's a bug we should fix.

Reply:
The documentation is wrong for all 3. In this PR, we should just update the documentation to be correct. We can do another change after to actually implement what this comment claims our code does.

Reply:
Good point. @Tianyi-W ,

You need to test with setting this as well:

```
encryption: dynamodb.TableEncryption.CUSTOMER_MANAGED
```

And see what happens. This @default tag should describe the default behavior for the `encryptionKey` property both when `encryption` is undefined and when `encryption` is `CUSTOMER_MANAGED`.